src/imagestyle.js

```diff
@@ -42,7 +42,7 @@ export default class ImageStyle extends Plugin {
  * The default value is:
  *
  *		const imageConfig = {
- *			styles: [ 'imageStyleFull', 'imageStyleSide' ]
+ *			styles: [ 'imageStyle:full', 'imageStyle:side' ]
```

Comment:
Well, this is odd. Below the name is set to `full` and here you use `imageStyle:full`. So I think that you wanted to change this but forgot at some point :D